lib/measured/measurable.rb

```diff
@@ -54,7 +54,7 @@ def <=>(other)
   class << self
 
     def conversion
-      @conversion ||= Measured::Conversion.new
+      raise "Measurable defined without a conversion"
```

Comment:
Nitpick, but this could be clearer.

```ruby
"Measurable does not have a `conversion` object. You cannot directly subclass `Measurable`. Instead, build a new unit system by calling `Measured.build`."
```